proposals/2021-01-stargate-upgrade-b/proposal.json

```diff
@@ -0,0 +1,7 @@
+
+{
+    "title":"Cosmos Stargate Hub Upgrade Proposal 2b: Good Things Come to Those Who Wait.",
+    "description":"Proposal to complete the Stargate upgrade, halt `cosmoshub-3` at 06:00 UTC on Jan 28th, export the state and start `cosmoshub-4` based on gaia 4.0.\n\n Gaia Commit hash: \n 2bb04266266586468271c4ab322367acbf41188f  Proposal details can be found on \n github: https://github.com/cosmos/governance/pull/13 \n ipfs: https://cloudflare-ipfs.com/ipfs/XX \n sia: https://siasky.net/XX \n  ",
```

Comment:
Parts of description don't match the actual proposal above, e.g, Jan 28th halt. Some invalid urls.